.circleci/docker/common/install_android.sh

```diff
@@ -45,52 +47,30 @@ export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
 # Installing android sdk
 # https://github.com/circleci/circleci-images/blob/staging/android/Dockerfile.m4
 
-_sdk_version=sdk-tools-linux-3859397.zip
+_tmp_sdk_zip=/tmp/android-sdk-linux.zip
 _android_home=/opt/android/sdk
 
 rm -rf $_android_home
 sudo mkdir -p $_android_home
-curl --silent --show-error --location --fail --retry 3 --output /tmp/$_sdk_version https://dl.google.com/android/repository/$_sdk_version
-sudo unzip -q /tmp/$_sdk_version -d $_android_home
-rm /tmp/$_sdk_version
+curl --silent --show-error --location --fail --retry 3 --output /tmp/android-sdk-linux.zip $_https_amazon_aws/android-sdk-linux.zip
```

Comment:
Shouldn't this be 
```suggestion
curl --silent --show-error --location --fail --retry 3 --output /tmp/android-sdk-linux.zip $_https_amazon_aws/sdk-tools-linux-3859397.zip
```

Reply:
Here I did a change how we install it, 

it was hard to just with unpacking archives to replicate all sdk state that should be equivalent to:

```
unzip sdk-tools-linux-3859397.zip
sdkmanager \
  "tools" \
  "platform-tools" \
  "emulator"

sdkmanager \
  "build-tools;28.0.3" \
  "build-tools;29.0.2"

sdkmanager \
  "platforms;android-28" \
  "platforms;android-29"
```

I zipped that state from my devserver to android-sdk-linux.zip
If we want to support naming how it was packed, probably it should be something like
android-sdk-linux-tools3859397-build-tools2803-2902-platforms28-29.zip